pages/docs/features/inngest-functions/steps-workflows/fetch.mdx

```diff
@@ -104,14 +104,8 @@ By default, using Inngest's `fetch` retains all the functionality of requests ma
 
 <CodeGroup>
 ```ts {{ title: "TypeScript" }}
-import { fetch as inngestFetch } from 'inngest';
 import { generateText } from 'ai';
-import { createAnthropic } from '@ai-sdk/anthropic';
-
-// The AI SDK's createAnthropic objects can be passed a custom fetch implementation
-const anthropic = createAnthropic({
```

Comment:
@Benanna2019 we need to keep these lines as the purpose of this example is to show how to provide Inngest's `fetch()` to the Vercel AI SDK